include/fastdds/rtps/history/History.h

```diff
@@ -225,6 +235,10 @@ class History
     //!Print the seqNum of the changes in the History (for debuggisi, mng purposes).
```

Comment:
I'm not going to ask for a change due to this, but if another commit is necessary for some reason, can we take the opportunity to correct this documentation here?